### PR TITLE
adding and configuring pre-commit fix #51

### LINKED ISSUE #51: Add pre-commit to azkaban-cli
### Overview of the issue
Add pre-commit to the project, with black and flake8 set up.

### Motivation for or Use Case
I believe that standardizing the project code style is a good practice, so we can use [black](https://github.com/psf/black) and flake8, a useful tool for this purpose is [pre-commit](https://pre-commit.com/). I would like to configure this tool in the project as part of my contributions to hacktober fest.

### PR DESCRIPTION
**Pull request by LeonardoFurtado**
Additionally I added the black badge in the readme and changed the travis badge to a "prettier" one. Waiting for your feedback :)

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401, F821
+max-line-length = 79
```
Comment (aguimaraesduarte) [inline .flake8]: Should we use max-line-length of 119 instead? I know PEP8 says 79, but it always feels so short... @ggalves what's your opinion on this?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[tool.black]
+line-length = 79
```
Comment (aguimaraesduarte) [inline pyproject.toml]: Same here @ggalves

File: .flake8 (hunk 1 above)
Comment (ggalves) [reply .flake8]: yeah, we usually use 120 rather than 80. Can you change it @LeonardoFurtado ?

File: pyproject.toml (hunk 2 above)
Comment (ggalves) [reply pyproject.toml]: agree

File: .flake8 (hunk 1 above)
Comment (LeonardoFurtado) [reply .flake8]: Yep!

Comment (LeonardoFurtado) [reply .flake8]: Done!

**Pull request merged**